pkg/endpoint/endpoint_test.go

```diff
@@ -0,0 +1,26 @@
+package endpoint
+
+import (
+	"context"
+	"testing"
+)
+
+func TestListen(t *testing.T) {
+	var config Config
+	var ctx context.Context
```

Comment:
```suggestion
	var (
		config Config
		ctx context.Context
	)
```
Just looks more idiomatic this way